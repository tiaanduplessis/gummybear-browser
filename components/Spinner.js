import React from 'react'
import { View, ActivityIndicator, StyleSheet } from 'react-native'

import colors from '../constants/colors'

const Spinner = () => (
  <View style={styles.container}>
    <ActivityIndicator size='large' color={colors.gray} />
  </View>
)

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
    backgroundColor: colors.offWhite
  }
})

export default Spinner
