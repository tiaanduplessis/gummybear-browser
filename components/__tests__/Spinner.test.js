import 'react-native'
import React from 'react'
import Spinner from '../Spinner'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
    const tree = renderer.create(<Spinner />).toJSON()
    expect(tree).toMatchSnapshot()
})
