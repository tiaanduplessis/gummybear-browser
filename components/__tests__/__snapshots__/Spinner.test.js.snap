// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F7F8F8",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#757575"
    hidesWhenStopped={true}
    size="large"
  />
</View>
`;
